fedimint-testing/src/ln/mock.rs

```diff
@@ -162,4 +162,11 @@ impl ILnRpcClient for FakeLightningTest {
     ) -> Result<EmptyResponse, LightningRpcError> {
         Ok(EmptyResponse {})
     }
+
+    async fn update_fedimint_scids(
```

Comment:
nit: I don't think the word fedimint does much for us here. I would prefer `update_target_scids`

Reply:
I changed it across the board to just `update_scids`